src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -70,6 +70,36 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Platform manifest needs to include all System*Native items from 3.1 release. This target adds
+    them explicitly, to fill the gap that appeared when the names of these libraries were changed
+    in 5.0 to conform to a specific Android naming convention. The library name change was a fix
+    for https://github.com/dotnet/runtime/issues/33118. For more details of the issue and fix see
+    https://github.com/dotnet/runtime/issues/33450.
+  -->
+  <ItemGroup>
```

Comment:
I think that as a convention, items/props are normally before the targets in the file unless there's some unusual ordering reason to do otherwise (like having to override some props defined in the imported targets file).

Reply:
Updating this with new commit.